src/internal/metrics/init-awsui-versions.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// not using `declare global {}` to avoid polluting customers' typings with this info
+interface CustomWindow extends Window {
+  awsuiVersions?: { components?: string[] };
+}
+declare const window: CustomWindow | undefined;
+
+// expose version info, so it can be checked using the browser devtools
+export function initAwsUiVersions(packageVersion: string) {
+  if (typeof window !== 'undefined') {
+    if (!window.awsuiVersions) {
+      window.awsuiVersions = {};
+    }
+    if (!window.awsuiVersions.components) {
```

Comment:
You probably want to make `components` part configurable too. To allow `{components: [], boards: []}` structure

Reply:
Oh, I kind of thought we'd like to keep components for both. But makes sense as the package versions can be different - will update 👍